cmd/helm-operator/main.go

```diff
@@ -16,17 +16,14 @@ package main
 
```

Comment:
Is not missing remove from `cmd/ansible-operator/main.go ` as well? 
Also, should we not add the fragment here with TBD?

Reply:
will be doing separate PR for Ansible, and regarding Fragment I had a question if we are adding individual fragments for removal PRs. As in the meeting, some one was saying about doing one big Fragment with all PRs. I Just need clarification on that.

Reply:
we can add many fragments in the same pr.

Reply:
Yeah, we should add individual removal fragments. Those require migration sections, but we're doing this:
```yaml
migration:
  header: <what got removed>
  body: TBD
```

Then when we go to do the release, we'll run the generator and then rewrite the release notes, using what's there to remind us what to cover.

Reply:
okay, Ill follow above template, and add fragment for this PR.

Reply:
e.g https://github.com/operator-framework/operator-sdk/pull/3414/files